### PR TITLE
Сизова -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by ksenya-sizova**

### CODE REVIEW

**Comment by ksenya-sizova** [comment]
@Nikita-Mitrokhin, @YuferevKirill, please review my changes

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/SizovaKsenyaTest.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        assertEquals(complexNumber.getImaginary(), 0, delta);
+    }
+
+    @Test
+    public void canCreateComplexNumberWithValues() {
+        ComplexNumber complexNumber = new ComplexNumber(1, 2);
+
+        // Assert
+        assertEquals(complexNumber.getReal(), 1, delta);
+        assertEquals(complexNumber.getImaginary(), 2, delta);
```
Comment (Nikita-Mitrokhin) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/SizovaKsenyaTest.java]: You use a lot of magic numbers, like here, fix it please.
it should look something like this:

        // Arrange
        ComplexNumber complexNumber = new ComplexNumber(1, 2);
        var expectedRealPart = 1;
        var expectedImagPart = 2;

        // Assert
        assertEquals(complexNumber.getReal(), expectedRealPart, delta);
        assertEquals(complexNumber.getImaginary(), expectedImagPart, delta);

**Comment by ksenya-sizova** [comment]
@Nikita-Mitrokhin, @YuferevKirill, please review my code after minor fix. Thanks!

**Comment by Nikita-Mitrokhin** [comment]
:+1:

**Comment by YuferevKirill** [comment]
👍

**Comment by ksenya-sizova** [comment]
@kirill-korniakov

**Pull request merged**